resources/config.edn

```diff
@@ -0,0 +1,76 @@
+{:datasets [{:indicator-id "23"
+             :indicator-field "Indicator value"
+             :conditions [{:field "Level description" :value "England"}]
```

Comment:
which makes me worry more about them not matching up at some point. downcasing everything means we can handle "Indicator value" and "Indicator Value" the same.